### ISSUE TITLE
Write inverse method for AsSemigroup (for a function and a digraph)

### ISSUE THREAD

**Issue by ChristopherRussell** [gap-packages/Digraphs#137]
This is a to-do. I should do this but I won't be doing it immediately so this issue can make me accountable in the future.

There should be an method `AsDigraph` which converts a semilattice semigroup to a digraph which is the inverse function to `AsSemigroup`. There should probably also be a method `IsomorphismSemigroup` which is a map from the vertices of a join semilattice digraph `digraph` to the output of `AsSemigroup(digraph)`.

**Comment by wilfwilson** [comment]
Yeah I thought about this too. It is a bit weird having a semigroup be isomorphism to a digraph, but I think it's fair enough.

**Comment by james-d-mitchell** [comment]
Is this addressed by PR #161?

**Comment by flsmith** [comment]
No, it's asking for the opposite method (semigroup -> digraph)

**Comment by james-d-mitchell** [comment]
Aw...
